### PR TITLE
Add possibility to manage which screen menu items will be shown for each type of building

### PR DESCRIPTION
**Pull request by javasia**

### CODE REVIEW

**Comment by javasia** [comment]
[3SP - Add possibility to manage which screen menu items will be shown for each type of building](https://trello.com/c/wgaSj6q1/125-3sp-add-possibility-to-manage-which-screen-menu-items-will-be-shown-for-each-type-of-building)

File: frontend/admin_panel/src/components/SidebarMenu/ScreenMenuItems/MenuItemMultipleSelect.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    fontWeight:
+      that.state.name.indexOf(name) === -1
+        ? that.props.theme.typography.fontWeightRegular
+        : that.props.theme.typography.fontWeightMedium
+  }
+}
+
+class MultipleSelect extends React.Component {
+  state = {
+    name: []
```
Comment (zozich) [inline frontend/admin_panel/src/components/SidebarMenu/ScreenMenuItems/MenuItemMultipleSelect.js]: `name` cannot be an array (by it's name). Call it `names` or `items`

File: frontend/admin_panel/src/components/SidebarMenu/ScreenMenuItems/MenuItemMultipleSelect.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  };
+
+  handleChange = event => {
+    this.setState({ name: event.target.value })
+  };
+
+  handleChangeMultiple = event => {
+    const { options } = event.target
+    const value = []
+    for (let i = 0, l = options.length; i < l; i += 1) {
```
Comment (zozich) [inline frontend/admin_panel/src/components/SidebarMenu/ScreenMenuItems/MenuItemMultipleSelect.js]: 1. Won't `event.target.value` work in this case as well?
2. If not, rework in functional style without using `for` loop
3. I don't see any usages of this function. Delete if not needed

**Comment by javasia** [comment]
[Create page for managing types of buildings with possibility to manage screen menu items per each type of buildings](https://trello.com/c/RTj9d0Zl/154-create-page-for-managing-types-of-buildings-with-possibility-to-manage-screen-menu-items-per-each-type-of-buildings)

**Pull request merged**